src/WebApi/WebApi/UseCases/v1/Account/Login/AccountController.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System.Linq;
+using System.Threading.Tasks;
+using Application.UseCases.User.CreateUser;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authentication.Cookies;
+using Microsoft.AspNetCore.Authentication.Google;
+using Microsoft.AspNetCore.Mvc;
+
+namespace WebApi.UseCases.v1.User.Login;
+
+[ApiVersion("1.0")]
+[Route("api/v{version:apiVersion}/[controller]/[action]")]
+[ApiController]
+public class AccountController : ControllerBase, IOutputPort
+{
+    private IActionResult _viewModel;
+
+    private readonly IEnsureCreatedUserUseCase _useCase;
+
+    public AccountController(IEnsureCreatedUserUseCase ensureCreatedUser)
+    {
+        _useCase = ensureCreatedUser;
+    }
+
+    void IOutputPort.Invalid()
+    {
+        _viewModel = BadRequest();
+    }
+
+    void IOutputPort.Ok()
+    {
+        _viewModel = Ok();
+    }
+
+    [HttpGet(Name = "Login")]
+    public async Task Login()
```

Comment:
There are 2 methods in one controller could we split it? 🤔

Reply:
It is closely related to Login method. Does it really make sense?